base/fastmath.jl

```diff
@@ -124,6 +125,38 @@ function make_fastmath(symb::Symbol)
 end
 make_fastmath(expr) = expr
 
+"""
+@fastmath expr
```

Comment:
>Actually, this should be a code block (indent it) 

Just indent everything inside the triple quotes, you mean?

> and this should be added to the doc/stdlib/

Sorry, I don't understand what you mean. Like, there should be a page for this in the official docs? Like a new markdown file?